### PR TITLE
feat(playwright): install browsers by default

### LINKED ISSUE #977: Add ability to run the install from the CreateAsync method
If we have a parameter `bool installIfNeeded` we could avoid having two lines with the `InstallAsync` and `CreateAsync`.

### PR DESCRIPTION
**Pull request by kblok**
closes #977 

This will be a breaking change for users using the dotnet tool.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,15 +36,17 @@ dotnet tool install playwright-sharp-tool -g
 playwright-sharp install-browsers
 ```
 
-By running these two commands, you can avoid having the `await Playwright.InstallAsync();` line in your code.
+If you runthese two commands, you can call `CreateAsync` passing `installIfNeeded` in false.
```
Comment (Meir017) [inline README.md:39]:
```suggestion
If you run these two commands, you can call `CreateAsync` passing `installIfNeeded` in false.
```

File: docfx_project/documentation/index.md [hunk 2]
```diff
@@ -36,15 +34,17 @@ dotnet tool install playwright-sharp-tool -g
 playwright-sharp install-browsers
 ```
 
-By running these two commands, you can avoid having the `await Playwright.InstallAsync();` line in your code.
+If you runthese two commands, you can call `CreateAsync` passing `installIfNeeded` in false.
```
Comment (Meir017) [inline docfx_project/documentation/index.md:37]:
```suggestion
If you run these two commands, you can call `CreateAsync` passing `installIfNeeded` in false.
```

File: src/PlaywrightSharp/Playwright.cs [hunk 3]
```diff
@@ -100,20 +100,27 @@ internal Playwright(IChannelOwner parent, string guid, PlaywrightInitializer ini
         /// <param name="debug">Enables the playwright driver log. Pass `pw:api` to get the Playwright API log.
         /// It is a shortcut to the DEBUG=pw:api environment variable.
         /// </param>
+        /// <param name=installIfNeeded">Call <see cref="InstallAsync(string, string)"/> before starting the driver</param>
```
Comment (Meir017) [inline src/PlaywrightSharp/Playwright.cs:103]:
```suggestion
        /// <param name=installIfNeeded">Call <see cref="InstallAsync(string, string)"/> before starting the driver.</param>
```

**Comment by kblok** [comment]
We are going to invest in a better dotnet tool.

**Pull request closed without merging**